### ISSUE TITLE
BernoulliWithSigmoidProbs is obsolete

### ISSUE THREAD

**Issue by jonasrauber** [tensorflow/tensorflow#9485]
I think `tf.contrib.distributions.BernoulliWithSigmoidProbs` can be removed, because `Bernoulli` itself has a `logits` parameter that does the exact same thing, afaik. If you agree, I can make a pull-request if necessary.

**Comment by aselle** [comment]
@ebrevdo, could you comment on this.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assigneee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by ebrevdo** [comment]
Yes; this looks safe to delete!

**Comment by tensorflowbutler** [comment]
A member of the TensorFlow organization has replied after the stat:awaiting tensorflower label was applied.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by yongtang** [comment]
Added PR #16846 for it.
